server/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -197,7 +198,7 @@ public void testClearIndexDelete() {
     }
 
     public void testCloseSearchContextOnRewriteException() {
-        createIndex("index");
+        createIndex("index", Settings.builder().put("index.refresh_interval", -1).build());
```

Comment:
I think a brief comment here would be useful, as it's not obvious why we would disable refresh for this test.